benchmarks/canonicalize_test.py

```diff
@@ -13,7 +13,7 @@ def main():
     time = 0
     time_canonicalize_url = 0
 
-    tar = tarfile.open("sites.tar.gz")
+    tar = tarfile.open("benchmarks/sites.tar.gz")
```

Comment:
Completely unrelated, but do you recall why we don't we use the same chromium urls here as well?